### PR TITLE
 Allow users to specify a directory for additonal devices

### PR DESCRIPTION
**Pull request by rhatdan**
Podman will search through the directory and will add any device
nodes that it finds.  If no devices are found we return an error.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: pkg/spec/config_linux.go [hunk 1]
```diff
@@ -27,6 +31,52 @@ func Device(d *configs.Device) spec.LinuxDevice {
...
+	resolvedDevicePath := devs[0]
+	// check if it is a symbolic link
+	if src, err := os.Lstat(resolvedDevicePath); err == nil && src.Mode()&os.ModeSymlink == os.ModeSymlink {
+		if linkedPathOnHost, err := filepath.EvalSymlinks(resolvedDevicePath); err == nil {
+			resolvedDevicePath = linkedPathOnHost
+		}
+	}
+	st, err := os.Stat(resolvedDevicePath)
+	if err != nil {
+		return errors.Wrapf(err, "can not stat %s", devicePath)
```
Comment (TomSweeneyRedHat) [inline pkg/spec/config_linux.go]: nit s/can not/cannot

**Comment by TomSweeneyRedHat** [comment]
LGTM, but would like another head nod before merge

**Comment by rhatdan** [comment]
bot, retest this please

**Pull request merged**